tensorflow/core/kernels/cwise_ops_test.cc

```diff
@@ -25,11 +25,12 @@ namespace tensorflow {
 
 // Creates a Graph which applies a unary "func" on a 3D float tensor
 // of "num" elements.
```

Comment:
while you're here -- could you please update the comment to say 'on a 3D tensor of type T with "num" elements' ?